xbmc/windowing/gbm/drm/DRMUtils.h

```diff
@@ -89,6 +90,7 @@ class CDRMUtils
   void PrintDrmDeviceInfo(drmDevicePtr device);
 
   int m_renderFd;
+  const char* m_renderDevicePath;
```

Comment:
```suggestion
  const char* m_renderDevicePath{};
```

better not to have it completely uninitialized

Reply:
thanks 😄